crates/types/src/lib.rs

```diff
@@ -45,14 +49,18 @@ mod sensor_data;
 mod sole_pressure;
 mod sonar_obstacle;
 mod sonar_values;
+mod step_adjustment;
 mod step_plan;
 mod support_foot;
 mod walk_command;
 mod whistle;
 mod world_state;
 
+// TODO: convert all "mod" to "pub mod"
```

Comment:
Added to this PR's description. I will not do this now because it will generate a lot of changes. Maybe I will do this later when all modules are ported and we can do such an disruptive change.